Expensify/App
# Improve Chat Switch timing in Grafana so it waits for all messages to layout

### Details
While A/B testing changes to Onyx cache stuff we discovered that this test does not actually measure the time it takes to tap on a chat and see the entire contents of the chat rendered. What we were testing is the time it takes the `ReportActionsView` to mount, but it might have no chat history available to it yet.

This new test will time how long it takes for all the items to render by debouncing a call passed to the `onLayout` of the view enclosing the chat content so it will record the "end" event once no one has called it after 1 second. We'll also then need to offset the timing by 1 second so that we get the true result.

This will give us a better sense of when the report is "ready to view".

**Note:** This test has one downside at the moment which is that @kidroca has pointed out that `VirtualizedList` may be configured incorrectly and rendering many more items than necessary see issue [here](https://github.com/Expensify/Expensify.cash/issues/2629). Still I think it's valuable to do this test now so we can understand when a report is actually done doing it's "work" to render the UI.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/2683

### Tests
1. Switch to a chat
2. Wait for the chat messages to load
3. Verify the timing log eventually shows with the time it took for all chat messages to render

### QA Steps
No QA

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
